README.md

```diff
@@ -1,11 +1,264 @@
+            
+        val defaults = service.getDefaults()
+        println("Project name: ${defaults.name}")
+        println("Project group: ${defaults.groupId}")
+        println("Project version: ${defaults.version}")
+    }
+}
+```
+
+### Option 2: Using build.gradle.kts
+
+Apply the plugin in your root `build.gradle.kts`:
+
+```kotlin
+plugins {
+    id("io.github.yonggoose.kotlin-pom-gradle-project") version "0.1.0"
+}
+
+rootProjectPom {
+    groupId = "io.github.yonggoose"
+    artifactId = "my-project"
+    version = "1.0.0"
+    
+    name = "My Project"
+    description = "A sample project"
+    url = "https://github.com/YongGoose/my-project"
+    
+    licenses {
+        license {
+            licenseType = "Apache-2.0"
+        }
+    }
+    
+    organization {
+        name = "YongGoose"
+        url = "https://github.com/YongGoose"
+    }
+
+    developers {
+        developer {
+            id = "yonggoose"
+            name = "Yongjun Hong"
+            email = "yongjunh@apache.org"
+        }
+    }
+    
+    inceptionYear = "2025"
+    
+    mailingLists {
+        mailingList {
+            name = "Developers"
+            subscribe = "dev-subscribe@example.org"
+            unsubscribe = "dev-unsubscribe@example.org"
+            post = "dev@example.org"
+            archive = "https://example.org/archive"
+        }
+    }
+    
+    issueManagement {
+        system = "GitHub"
+        url = "https://github.com/YongGoose/my-project/issues"
+    }
+    
+    scm {
+        url = "https://github.com/YongGoose/my-project"
+        connection = "scm:git:git@github.com:YongGoose/my-project.git"
+        developerConnection = "scm:git:git@github.com:YongGoose/my-project.git"
+    }
+}
+```
+
+In your submodule's `build.gradle.kts`, apply the plugin and override settings as needed:
+
+```kotlin
+plugins {
+    id("io.github.yonggoose.kotlin-pom-gradle-project") version "0.1.0"
+}
+
+// Override only what's needed
+projectPom {
+    artifactId = "my-submodule"
+    description = "A submodule of my project"
+    // Any other properties you need to override
+}
+
+// Access the merged values
+tasks.register("printProjectInfo") {
+    doLast {
+        val pom = project.extensions.extraProperties.get("mergedDefaults") as OrganizationDefaults
+        println("Project name: ${pom.name}")
+        println("Project group: ${pom.groupId}")
+        println("Project version: ${pom.version}")
+    }
+}
+```
+<details>
+<summary>Data Structure</summary>
+
+- Basic information: groupId, artifactId, version, name, description, url, inceptionYear  
+- Organization information: name, url  
+- License details: licenseType  
+- Developers: id, name, email, url, organization, organizationUrl, timezone  
+- Mailing lists: name, subscribe, unsubscribe, post, archive  
+- Issue management: system, url  
+- SCM: connection, developerConnection, url
+
+</details>
+
+## 2. Artifact Signing & POM Validation
+
+### Artifact Signing Verification
```

Comment:
Actually, the change verifies the POMs and metadata, but not artifact signing. The description and texts need to be updated